bower.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "react-motion",
+  "version": "0.0.2",
+  "homepage": "https://github.com/chenglou/react-motion",
+  "authors": [
+    "chenglou"
+  ],
+  "description": "A spring that solves your animation problems.",
+  "main": ["lib/Spring.js"],
+  "dependencies": {
+    "react": "~0.13.2"
```

Comment:
Can you use >= for this? I used to keep getting pinged to upgrade my other libraries because I used `~` and their React version is different